src/nvim/main.c

```diff
@@ -1174,10 +1173,10 @@ static void command_line_scan(mparm_T *parmp)
               p = xmalloc(STRLEN(a) + 4);
               sprintf((char *)p, "so %s", a);
               parmp->cmds_tofree[parmp->n_commands] = TRUE;
-              parmp->commands[parmp->n_commands++] = p;
-            } else
-              parmp->commands[parmp->n_commands++] =
-                (char_u *)argv[0];
+              parmp->commands[parmp->n_commands++] = (char *)p;
```

Comment:
Lools like `p` is defined just a few lines above. Can it be localized and declared as a `char *`?